Ginger/GingerCoreCommon/Repository/NewRepositorySerializer.cs

```diff
@@ -108,6 +108,7 @@ public string SerializeToString(RepositoryItemBase ri)
                         xml.WriteEndDocument();
                     }
                     string result = Encoding.UTF8.GetString(output.ToArray());
+                    //result = Regex.Replace(result, "&#x([0-8BCEFbcef]|1[0-9A-Fa-f]);|[\x01-\x08\x0B\x0C\x0E\x0F\u0000-\u0008\u000B\u000C\u000E-\u001F]", "");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5038055)